DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -192,7 +192,7 @@ public class MockPodCommManager: PodCommManagerProtocol {
                 reservoirUnitsRemaining: 1023,
                 timeElapsedSinceActivation: 2,
                 activationTime: activation)
-            self.podCommState = .alarm(MockPodAlarm.occlusion)
+            self.podCommState = .active // .alarm(MockPodAlarm.occlusion)
```

Comment:
[nit] commented-out code?  Was that intentional?

Reply:
Yes; thought I'd leave it there as an example of a way to put the mock pod in an alarm state.